### PR TITLE
{chem}[foss/2020b] pdb2pqr v3.1.0 w/ Python 3.8.6

### PR DESCRIPTION
**Pull request by ThomasHoffmann77**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/p/pdb2pqr/pdb2pqr-3.1.0-foss-2020b.eb [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        'patches': ['pdb2pqr-3.1.0_mmcif_pdbx_2.0.0_remove_versioneer.patch'],
+        'sources': 'mmcif_pdbx-%(version)s.tar.gz',
+        'checksums': [
+            'ffd76b08f0215d052fcf3015b8bac927cdf369ca60e4e26974f8aec99ca66986',  # mmcif_pdbx-2.0.0.tar.gz
+            # pdb2pqr-3.1.0_mmcif_pdbx_2.0.0_remove_versioneer.patch
+            'fbf8c34361aa0c4060eff071be781fee73db494aa7d1a63424d92986370ac1da',
+        ],
+    }),
+    (name, version, {
+        'checksums': ['a6f5e2eefbd8bd2a072a89ee4026977de02dadd56e921095afac88d596b63820'],
```
Comment (Micket) [inline easybuild/easyconfigs/p/pdb2pqr/pdb2pqr-3.1.0-foss-2020b.eb:36]: No sources?

File: easybuild/easyconfigs/p/pdb2pqr/pdb2pqr-3.1.0_mmcif_pdbx_2.0.0_remove_versioneer.patch [hunk 2]
```diff
@@ -0,0 +1,42 @@
+diff -ru mmcif_pdbx-2.0.0/MANIFEST.in mmcif_pdbx-2.0.0_patched/MANIFEST.in
```
Comment (Micket) [inline easybuild/easyconfigs/p/pdb2pqr/pdb2pqr-3.1.0_mmcif_pdbx_2.0.0_remove_versioneer.patch:1]: A short description of what the patch does, reference to issue, and author are appreciated for patches

**Review by Micket — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/p/pdb2pqr/pdb2pqr-3.1.0-foss-2020b.eb [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+exts_list = [
+    ('propka', '3.4.0', {
+        'souces': '%(name)s-%(version)s.tar.gz',
+        'checksums': ['f19c2e145faa7eab000ce056a9058b8a30adba6970705047bb4fb7ba7570b020'],
+    }),
+    ('mmcif-pdbx', '2.0.0', {
+        'modulename': 'pdbx',
+        # Use hardcoded version and remove problematic versioneer import:
+        'patches': ['pdb2pqr-3.1.0_mmcif_pdbx_2.0.0_remove_versioneer.patch'],
+        'sources': 'mmcif_pdbx-%(version)s.tar.gz',
```
Comment (boegel) [inline easybuild/easyconfigs/p/pdb2pqr/pdb2pqr-3.1.0-foss-2020b.eb:30]: @ThomasHoffmann77 Same here

```python
'sources': ['mmcif_pdbx-%(version)s.tar.gz'],
```

File: easybuild/easyconfigs/p/pdb2pqr/pdb2pqr-3.1.0-foss-2020b.eb [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+dependencies = [
+    ('Python', '3.8.6'),
+    ('SciPy-bundle', '2020.11'),
+]
+
+use_pip = True
+exts_list = [
+    ('propka', '3.4.0', {
+        'souces': '%(name)s-%(version)s.tar.gz',
```
Comment (boegel) [inline easybuild/easyconfigs/p/pdb2pqr/pdb2pqr-3.1.0-foss-2020b.eb:23]: @ThomasHoffmann77 This should be a list:

```python
'souces': ['%(name)s-%(version)s.tar.gz'],
```

File: easybuild/easyconfigs/p/pdb2pqr/pdb2pqr-3.1.0-foss-2020b.eb [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+        'patches': ['pdb2pqr-3.1.0_mmcif_pdbx_2.0.0_remove_versioneer.patch'],
+        'sources': 'mmcif_pdbx-%(version)s.tar.gz',
+        'checksums': [
+            'ffd76b08f0215d052fcf3015b8bac927cdf369ca60e4e26974f8aec99ca66986',  # mmcif_pdbx-2.0.0.tar.gz
+            # pdb2pqr-3.1.0_mmcif_pdbx_2.0.0_remove_versioneer.patch
+            '0e48f4cc680e7e6e07670ee8380bee09a6834afb1399f83748d6c9fe9d92301f',
+        ],
+    }),
+    (name, version, {
+        'souces': '%(name)s-%(version)s.tar.gz',
```
Comment (boegel) [inline easybuild/easyconfigs/p/pdb2pqr/pdb2pqr-3.1.0-foss-2020b.eb:38]: Same here: `[...]`

**Review by boegel — CHANGES_REQUESTED** [review]

**Comment by boegel** [comment]
closing this since `foss/2020b` is deprecated, see https://docs.easybuild.io/policies/toolchains

**Pull request closed without merging**